source/beamline/AMSingleEnumeratedControl.cpp

```diff
@@ -108,9 +111,14 @@ bool AMSingleEnumeratedControl::addValueOption(int index, const QString &optionS
 	return result;
 }
 
-bool AMSingleEnumeratedControl::addValueOption(int index, const QString &optionString, double optionSetpoint)
+bool AMSingleEnumeratedControl::addValueOption(int index, const QString &optionString, double optionMin, double optionMax)
+{
+	return addValueOption(index, optionString, -1, optionMin, optionMax, true);
+}
+
+bool AMSingleEnumeratedControl::addValueOption(int index, const QString &optionString, double optionSetpoint, bool readOnly)
 {
-	return addValueOption(index, optionString, optionSetpoint, optionSetpoint, optionSetpoint);
+	return addValueOption(index, optionString, optionSetpoint, optionSetpoint, optionSetpoint, readOnly);
```

Comment:
Just want to make sure that for this overloaded `addValueOption()`, do we really want the `optionMin` == `optionMax` == `optionSetpoint`?